src/irr/CMakeLists.txt

```diff
@@ -0,0 +1,521 @@
+find_package(OpenGL REQUIRED)
```

Comment:
what is this file doing here? it was renamed and removed

Reply:
somethings gone bad during the merge, ill look at it in a moment